specs/mediawiki/v1/summary.yaml

```diff
@@ -0,0 +1,110 @@
+swagger: '2.0'
+info:
+  version: '1.0.0-beta'
+  title: MediaWiki Summary API
+  description: Page content summary API
+  termsofservice: https://github.com/wikimedia/restbase#restbase
+  contact:
+    name: Services
+    email: services@lists.wikimedia.org
+    url: https://www.mediawiki.org/wiki/Services
+  license:
+    name: Apache licence, v2
+    url: https://www.apache.org/licenses/LICENSE-2.0
+x-default-params:
+  domain: en.wikipedia.org
+paths:
+  /{module:page}/summary/{title}:
+    get:
+      tags:
+        - Page content
+      description: >
+        Returns the summary of the latest page content available in storage.
+        Currently the summary includes the text for the first several sentences and
+        the thumbnail URL.
+
+        Provide a `Cache-Control: no-cache` header to request the latest data.
+
+        Stability: [experimental](https://www.mediawiki.org/wiki/API_versioning#Experimental)
+      produces:
+        - application/json
+      parameters:
+        - name: title
+          in: path
+          description: The page title.
+          type: string
+          required: true
+      responses:
+        '200':
+          description: >
+            The summary for the given page
```

Comment:
Nit: no need to go into a separate row, it's short enough not to make jscs complain :)